csharp/test/ErrorTest.cs

```diff
@@ -32,5 +32,24 @@ public void BagSignature()
 
             verify.Should().Throw<SignatureException>();
         }
+
+        [Fact]
```

Comment:
Could you do a Theory with a few cases where a different header is missing each time? Looks like this is testing missing `tl_version` only

Reply:
Sure I'll take a look tomorrow